tests/integration/test_zos_jobs.py

```diff
@@ -32,6 +32,15 @@ def test_list_jobs_should_return_valid_spool_information(self):
         """Executing the list_jobs method should return a list of found jobs in JES spool."""
         command_output = self.jobs.list_jobs(owner=self.jobs_fixtures_json['TEST_JCL_OWNER'])
         self.assertIsInstance(command_output, list)
+    
+    def test_change_job_class(self):
+        """Execute the change_jobs_class should execute successfully."""
+        execution_output = self.jobs.submit_from_mainframe(self.jobs_fixtures_json['TEST_JCL_MEMBER'])
+        jobname = execution_output['jobname']
+        jobid = execution_output['jobid']
+        command_output = self.jobs.change_jobs_class(jobname, jobid, "A")
+        expected_class = self.jobs.get_job_status(jobname, jobid)
+        self.assertEqual(expected_class['class'], "A")
```

Comment:
Thanks for adding `TEST_JCL_CLASS` to the fixture JSON 🙂 

Can we replace "A" on lines 41 and 43 with the value of `TEST_JCL_CLASS` loaded from the fixture?

Reply:
The class name shouldn't come from `expected_output`, we want to change it to a different class - the one defined in `self.jobs_fixtures_json`